src/NServiceBus.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -55,7 +56,6 @@ public IConnection CreateConnection(string connectionName)
         {
             lock (lockObject)
             {
-                connectionFactory.ClientProperties["purpose"] = connectionName;
```

Comment:
That's correct. Setting the purpose value is redundant now that the connection name property is displayed properly in the RabbitMQ management UI.